Gemfile.lock

```diff
@@ -619,4 +619,4 @@ RUBY VERSION
    ruby 2.6.6p146
 
 BUNDLED WITH
-   1.17.3
+   2.2.19
```

Comment:
can we revert this change please? just to keep this PR focused on the Ruby 2.7.1 update and leave updating bundler in a separate PR which we can do after this one goes out :)

Reply:
Sure!